tests/libvmi/vmi.go

```diff
@@ -126,6 +126,14 @@ func WithSEV() Option {
 	}
 }
 
+func WithSEVAttestation() Option {
+	return func(vmi *kvirtv1.VirtualMachineInstance) {
+		startStrategy := v1.StartStrategyPaused
+		vmi.Spec.StartStrategy = &startStrategy
+		vmi.Spec.Domain.LaunchSecurity.SEV.Attestation = &v1.SEVAttestation{}
+	}
+}
+
```

Comment:
I wonder if we need to also initiate `LaunchSecurity` here (as `WithSEV()` does). WDYT?